### DISCUSSION TITLE
Dolby Atmos not supporting on Android 14

### DISCUSSION THREAD

**Discussion by NoelEldhoNommy** [reiryuki/Mi-Sound-Redmi-K40-Magisk-Module#9] [Show and tell]
https://t.me/androidryukimodsdiscussions/2618
Hey there I am using redmi note 10 pro Max (sweet) and I am running on a custom ROM with MIUI core v4.18 (cepheus) and I am running Android version 14 pixelOS kernel version 4.14.329-VantomKernel
#1 Sun Nov 19 08:21:54 CET 2023. I could not install the Dolby Atmos module because of report stack pointer ev not found. I have attached log file and the screenshot in the discription below.
![Screenshot_20231124-172352](https://github.com/reiryuki/Mi-Sound-Redmi-K40-Magisk-Module/assets/99970976/9833dffa-6ba6-4ae0-b663-1b3b9315e1e6)
[magisk_install_log_2023-11-24T17.25.35.log](https://github.com/reiryuki/Mi-Sound-Redmi-K40-Magisk-Module/files/13458712/magisk_install_log_2023-11-24T17.25.35.log)

**Comment by Vilez0** [comment]
same issue
